Swiftfin tvOS/Views/ItemView/CollectionItemView/CollectionItemContentView.swift

```diff
@@ -26,16 +27,28 @@ extension CollectionItemView {
                     .padding(.bottom, 50)
 
                 if viewModel.collectionItems.isNotEmpty {
-                    PosterHStack(
-                        title: L10n.items,
-                        type: .portrait,
-                        items: viewModel.collectionItems
-                    )
-                    .onSelect { item in
-                        router.route(to: \.item, item)
+
+                    ForEach(BaseItemKind.allCases, id: \.self) { sectionType in
+                        let sectionItems = viewModel.collectionItems.filter { $0.type == sectionType }
```

Comment:
Instead, have the `collectionItems` be an `OrderedDictionary<BaseItemKind, [BaseItemDto]>`